Cargo.toml

```diff
@@ -287,3 +287,6 @@ opt-level = 2
 [profile.dist]
 inherits = "release"
 lto = "off"
+
+[features]
+trezor = []
```

Comment:
I think it should be enabled in CI

Reply:
this comment is still valid. We can't run tests on CI but we can at least build the code